### ISSUE TITLE
How to increase validation accuracy

### ISSUE THREAD

**Issue by diliphadoop742** [meownoid/face-identification-tpe#18]
Hello meownoid,

while training after 270 epochs we got validation accuracy of 0.0046, which is constant from starting and validation loss of 16.563. can you update the repository by changing train_cnn.py so that the model will provide better accuracy during training.

**Comment by meownoid** [comment]
Repository is now archived, closing all issues.
